docs/writing-tests/visual-testing.md

```diff
@@ -13,76 +14,113 @@ Ideal for verifying what the user sees: layout, color, size, and contrast. Story
+![Confirm UI changes in Storybook](./vta-changes-found.png)
 
-![Chromatic project first build](./chromatic-first-build-optimized.png)
+When you finish accepting changes as baselines in the addon, you're ready to push the code to your remote repository. This will sync baselines to the cloud for anyone who checks out your branch.
 
-## Catching UI changes
+![Accept UI changes in Storybook](./vta-changes-accepted.png)
 
-Each time you run Chromatic, it will generate new snapshots and compare them against the existing baselines. That’s ideal for detecting UI changes and preventing potential UI regressions.
+### Automate with CI
 
-For example, let's assume you're working on a component and you tweak the styling. When Chromatic is re-run, it will highlight the difference between the baseline and the updated component.
+The addon is designed to be used in tandem with CI. We recommend using the addon to check for changes during development and then running visual tests in CI as you get ready to merge.
 
-![Chromatic project second build](./chromatic-second-build-optimized.png)
+Changes you accept as baselines in the addon will get auto-accepted as baselines in CI so you don’t have to review twice.
 
-If the changes are intentional, accept them as baselines. Otherwise, deny them to prevent UI regressions.
+1. Add a step to your CI workflow to run Chromatic.
 
-Learn how to [integrate Chromatic UI Tests](https://www.chromatic.com/docs/?utm_source=storybook_website&utm_medium=link&utm_campaign=storybook) into your CI pipeline.
+   - [GitHub Actions](https://chromatic.com/docs/github-actions?ref=storybook_docs)
+   - [GitLab Pipelines](https://chromatic.com/docs/gitlab?ref=storybook_docs)
+   - [Bitbucket Pipelines](https://chromatic.com/docs/bitbucket-pipelines?ref=storybook_docs)
+   - [CircleCI](https://chromatic.com/docs/circleci?ref=storybook_docs)
+   - [Travis CI](https://chromatic.com/docs/travisci?ref=storybook_docs)
+   - [Jenkins](https://chromatic.com/docs/jenkins?ref=storybook_docs)
+   - [Azure Pipelines](https://chromatic.com/docs/azure-pipelines?ref=storybook_docs)
+   - [Custom CI provider](https://chromatic.com/docs/custom-ci-provider?ref=storybook_docs)
+
+2. Configure your CI to include environment variables to authenticate with Chromatic (project token).
+
+#### PR checks
+
+Once you successfully set up Chromatic in CI, your pull/merge requests will be badged with a UI Tests check. The badge notifies you of test errors or UI changes that need to be verified by your team. Make the check required in your Git provider to prevent accidental UI bugs from being merged.
+
+![PR badge for visual tests](./vta-prbadge-test.png)
 
 ---
 
-#### What’s the difference between visual tests and snapshot tests?
+### What’s the difference between visual tests and snapshot tests?
```

Comment:
@domyen — I realize you made this an H4 so that it had less visual prominence. But now that we have the "On this page" TOC, the incorrect heading level made this appear to be under the CI section. I viewed that confusion as worse than the visual aspect.

Also, I added a brief paragraph on visual tests, to more completely answer the actual question here. Feel free to tweak!

Reply:
Thanks for the explanation about the H4! 

Your paragraph looks great ❤️